Source/WebCore/platform/graphics/cocoa/IOSurface.mm

```diff
@@ -685,6 +692,30 @@ static IntSize computeMaximumSurfaceSize()
     m_colorSpace = surfaceColorSpace().value_or(DestinationColorSpace::SRGB());
 }
 
+#if HAVE(SUPPORT_HDR_DISPLAY)
+void IOSurface::setContentEDRHeadroom(float headroom)
+{
+    if (m_contentEDRHeadroom && headroom == m_contentEDRHeadroom)
+        return;
+
+    LOG_WITH_STREAM(HDR, stream << "IOSurface::setContentEDRHeadroom " << this << " " << headroom);
+    m_contentEDRHeadroom = headroom;
+    IOSurfaceSetValue(m_surface.get(), CFSTR("IOSurfaceContentHeadroom"), @(headroom));
```

Comment:
should we define `"IOSurfaceContentHeadroom"` in `IOSurfaceSPI.h` like the other IOSurface properties we use?

Reply:
Indeed we should, done!